packages/frontend-2/components/dashboard/TutorialCard.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <NuxtLink v-if="tutorial.url" :to="tutorial.url" external>
```

Comment:
+ Its wasteful to even manage the component in memory if you don't actually want it. This `v-if` should be handled by the parent component - it should filter out any tutorial objects w/o an URL.

But this may be moot - is it really the case that TutorialCard.url can be optional? Feels like thats a prop that would always be there

Reply:
@fabis94 This comes from an external API, and technically `url` can be empty, it doesn't happen in practice but the type of the url allows for it